docs/DICTIONARIES.md

```diff
@@ -12,6 +12,7 @@ This project uses `dictionaries` to unify props between different components.
 - [Size](#size)
 - [Color](#color)
 - [Validation](#validation)
+- [Align](#align)
```

Comment:
What about renaming it also here? The dictionary was renamed in code but not in the documentation.

Reply:
done